plugins/Microsoft.DocAsCode.Build.UniversalReference/BuildUniversalReferenceDocument.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.UniversalReference
+{
+    using System.Composition;
+
+    using Microsoft.DocAsCode.Build.Common;
+    using Microsoft.DocAsCode.Plugins;
+
+    [Export(nameof(UniversalReferenceDocumentProcessor), typeof(IDocumentBuildStep))]
+    public class BuildUniversalReferenceDocument : BuildReferenceDocumentBase
+    {
+        private readonly IModelAttributeHandler _handler =
+            new CompositeModelAttributeHandler(
+                new UniqueIdentityReferenceHandler(),
+                new MarkdownContentHandler()
+            );
+
+        public override string Name => nameof(BuildUniversalReferenceDocument);
+
+        #region BuildReferenceDocumentBase
+
+        protected override void BuildArticle(IHostService host, FileModel model)
```

Comment:
change to virtual?

---
In reply to: [114691552](https://github.com/dotnet/docfx/pull/1621#discussion_r114691552) [](ancestors = 114691552,114690938)